infra/charts/turing/templates/virtualservice-mlp.yaml

```diff
@@ -57,6 +57,18 @@ spec:
             host: {{ template "turing.merlin.name" . }}
             port:
               number: 8080
+    - match:
```

Comment:
How is this going to be used? I.e., how does Turing know when to call the `/api/xp/` endpoint?

Reply:
Good point, I think this will not be used, I've removed it.

Reply:
I didn't actually mean that it should be removed - my question was really, how will Turing use this, hinting at the absence of the glue.